modernizer-maven-plugin/src/main/java/org/gaul/modernizer_maven_plugin/ModernizerMojo.java

```diff
@@ -213,21 +214,36 @@ public void execute() throws MojoExecutionException {
             }
         }
 
-        File ignoreClassesFile = new File(
-            ModernizerAnnotationOutput.getOutputDir(outputDirectory),
-            ModernizerAnnotationOutput.IGNORE_CLASSES_FILE_NAME);
-        if (ignoreClassesFile.exists()) {
-            Collection<String> ignoreClasses =
-                readExclusionsFile(ignoreClassesFile.toString());
-            for (String ignoreClass : ignoreClasses) {
-                allIgnoreFullClassNamePatterns.add(
-                    Pattern.compile(ignoreClass));
-            }
+        String ignoreClassesFileName =
+            ModernizerAnnotationOutput.IGNORE_CLASSES_FILE_NAME;
+        Collection<String> ignoreClasses = getIgnoreElements(
+            ModernizerAnnotationOutput.getOutputDir(outputDirectory) + "/" +
```

Comment:
Is it possible to `new File(...)` here? It looks like this would be useful to store in a variable because you construct it many times.